packages/rancher-monitoring/66.7/rancher-monitoring/generated-changes/patch/Chart.yaml.patch

```diff
@@ -29,97 +27,18 @@
-   name: hardenedNodeExporter
-   repository: file://./charts/hardenedNodeExporter
--  version: 0.2.0
- - condition: k3sServer.enabled
-   name: k3sServer
-   repository: file://./charts/k3sServer
--  version: 0.2.0
- - condition: kubeStateMetrics.enabled
-   name: kube-state-metrics
-   repository: file://./charts/kube-state-metrics
--  version: 5.21.0
- - condition: kubeAdmControllerManager.enabled
-   name: kubeAdmControllerManager
-   repository: file://./charts/kubeAdmControllerManager
--  version: 0.2.0
- - condition: kubeAdmEtcd.enabled
-   name: kubeAdmEtcd
-   repository: file://./charts/kubeAdmEtcd
--  version: 0.2.0
- - condition: kubeAdmProxy.enabled
-   name: kubeAdmProxy
-   repository: file://./charts/kubeAdmProxy
--  version: 0.2.0
- - condition: kubeAdmScheduler.enabled
-   name: kubeAdmScheduler
-   repository: file://./charts/kubeAdmScheduler
--  version: 0.2.0
- - condition: prometheus-adapter.enabled
-   name: prometheus-adapter
-   repository: file://./charts/prometheus-adapter
--  version: 4.10.0
- - condition: nodeExporter.enabled
+@@ -57,10 +66,6 @@
    name: prometheus-node-exporter
    repository: file://./charts/prometheus-node-exporter
--  version: 4.37.1
+   version: 4.41.0
 -- condition: windowsMonitoring.enabled
 -  name: prometheus-windows-exporter
 -  repository: https://prometheus-community.github.io/helm-charts
--  version: 0.3.*
+-  version: 0.7.*
  - condition: rke2ControllerManager.enabled
    name: rke2ControllerManager
    repository: file://./charts/rke2ControllerManager
--  version: 0.2.0
- - condition: rke2Etcd.enabled
-   name: rke2Etcd
-   repository: file://./charts/rke2Etcd
--  version: 0.2.0
- - condition: rke2IngressNginx.enabled
-   name: rke2IngressNginx
-   repository: file://./charts/rke2IngressNginx
--  version: 0.2.0
- - condition: rke2Proxy.enabled
-   name: rke2Proxy
-   repository: file://./charts/rke2Proxy
--  version: 0.2.0
- - condition: rke2Scheduler.enabled
-   name: rke2Scheduler
-   repository: file://./charts/rke2Scheduler
--  version: 0.2.0
- - condition: rkeControllerManager.enabled
-   name: rkeControllerManager
-   repository: file://./charts/rkeControllerManager
--  version: 0.2.0
- - condition: rkeEtcd.enabled
-   name: rkeEtcd
-   repository: file://./charts/rkeEtcd
--  version: 0.2.0
- - condition: rkeIngressNginx.enabled
-   name: rkeIngressNginx
-   repository: file://./charts/rkeIngressNginx
--  version: 0.2.0
- - condition: rkeProxy.enabled
-   name: rkeProxy
-   repository: file://./charts/rkeProxy
--  version: 0.2.0
- - condition: rkeScheduler.enabled
-   name: rkeScheduler
-   repository: file://./charts/rkeScheduler
--  version: 0.2.0
- - condition: windowsExporter.enabled
-   name: windowsExporter
-   repository: file://./charts/windowsExporter
--  version: 0.3.1
- description: kube-prometheus-stack collects Kubernetes manifests, Grafana dashboards,
-   and Prometheus rules combined with documentation and scripts to provide easy to
```

Comment:
Why are these getting removed?

Reply:
This is a removal on a patch (diff), so what's actually being removed is the removal of the `version` field.


Not sure where the artifact of removing those versions came from, but it technically makes the syntax of the yaml invalid (and just leads to breaks at rebase). So without this part of the diff the patch of a previous rebase is more likely to work with minimal changes.